### DISCUSSION TITLE
trouble with tailwind

### DISCUSSION THREAD

**Discussion by koenarnold** [tailwindlabs/tailwindcss#13470] [Help] [answered]
Hey, so unfortunately I'm having some trouble getting tailwind to work for me. It's driving me a bit nuts because this should be pretty simple and easy but I've hit a wall and I'm not sure what to do. I followed the docs exactly but I haven't had any luck, I decided to make a new react+vite template and put the repo on github so hopefully someone can provide some guidance. 

![Screenshot 2024-04-07 at 9 07 21 PM](https://github.com/tailwindlabs/tailwindcss/assets/121586584/329b6b30-1267-417a-8109-44a195b27989)

I have tried many solutions; adding tailwindcss() to the plugins in vite.config.js, adding <script src="https://cdn.tailwindcss.com"></script> to my index.html, verifying I have the latest versions of each dependency, etc. Frankly I'm at a loss, any help would be much appreciated!

Repo Link: https://github.com/koenarnold/tailwind-issue

**Comment by wongjn** [comment, marked as answer]
What gives you the impression that Tailwind is not working? It doesn't seem you have used any Tailwind classes in any code covered by the `content` file globs. As soon as you do, the appropriate Tailwind class CSS rules should be generated into the compiled CSS.

**Reply by koenarnold** [reply]
Geez, honestly thank you. I don't know why I didn't bother to check and see if it actually even worked, I was getting thrown off by vscode saying "Unknown at rule @tailwind" and immediately got to troubleshooting. Just checked and it totally works, glad it was an easy fix but man I feel a bit foolish haha. Appreciate you taking the time to respond
